modules/_node-scripts/bundle/esbuild/bundleJavaScriptExports.mjs

```diff
@@ -65,6 +66,15 @@ async function bundle(
 		target: ['es2022'],
 	};
 
+	if (moduleName === 'ckeditor5') {
```

Comment:
@bryceosterhaus Can you think of any other place to put this? :thinking: Maybe in the global config? :thinking: 

It's not so horrible to put it here because in the end `node-scripts` is internal to `liferay-portal` but if we start following this path the code can become more difficult to maintain in the future...

Reply:
do we have anything in node-scripts to incorporate aliases from the project-level config?

Reply:
Yep, I figured that out. So, do we want to go with this and then allow aliases at project level, or shall we wait till this is a more general need (if at all)?

I'm a bit hesitant to allow this just for 1 project, but current solution looks difficult to maintain.

Perhaps node-scripts can keep a list of project which aliases are allowed, then only for those, apply what's in the config.

So anyone using this needs to enable the feature in node-scripts first and we keep track of the mechanism usages.

Reply:
> I would simply add a new alias field in node-scripts.config.js and use it.

Sounds good, I'll try it.